<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomFileDialog</class>
 <widget class="QDialog" name="CustomFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Open</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>14</y>
     <width>116</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Look in:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="CB_pathHistory">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>10</y>
     <width>201</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_navigateBackward">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>30</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Back</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../usr/os-bin/images/back.png</normaloff>../../../../usr/os-bin/images/back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_navigateForward">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>30</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Forward</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../usr/os-bin/images/forward.png</normaloff>../../../../usr/os-bin/images/forward.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_IconView">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>30</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Icon View</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../usr/os-bin/images/IconView.png</normaloff>../../../../usr/os-bin/images/IconView.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_ListView">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>30</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Detail View</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../usr/os-bin/images/listView.png</normaloff>../../../../usr/os-bin/images/listView.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_parentDirectory">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>11</y>
     <width>30</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>27</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>27</height>
    </size>
   </property>
   <property name="toolTip">
    <string>Parent Directory</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../usr/os-bin/images/arrow_up.png</normaloff>../../../../usr/os-bin/images/arrow_up.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="LW_SideBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>110</width>
     <height>250</height>
    </rect>
   </property>
  </widget>
  <widget class="QTreeView" name="TV_BrowseArea">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>420</width>
     <height>250</height>
    </rect>
   </property>
   <property name="rootIsDecorated">
    <bool>false</bool>
   </property>
   <property name="itemsExpandable">
    <bool>false</bool>
   </property>
   <property name="expandsOnDoubleClick">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QListView" name="LV_BrowseArea">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>420</width>
     <height>250</height>
    </rect>
   </property>
   <property name="movement">
    <enum>QListView::Static</enum>
   </property>
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="gridSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="viewMode">
    <enum>QListView::IconMode</enum>
   </property>
   <property name="uniformItemSizes">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>307</y>
     <width>544</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="horizontalSpacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="LB_FolderFile">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>File Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LE_selectedFilesFolders">
        <property name="minimumSize">
         <size>
          <width>420</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>420</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="LB_FileTypes">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>File of Types:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="CB_FilesTypes">
        <property name="minimumSize">
         <size>
          <width>420</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>420</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>348</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="PB_Cancel">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PB_Open">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
